pkg/cli/cmd/app/graph/graph_test.go

```diff
@@ -50,7 +50,7 @@ func Test_Validate(t *testing.T) {
 			},
 			ConfigureMocks: func(mocks radcli.ValidateMocks) {
 				mocks.ApplicationManagementClient.EXPECT().
-					ShowApplication(gomock.Any(), "test-app").
+					GetApplication(gomock.Any(), "test-app").
```

Comment:
The changes to the client API resulted in changes to most commands, but they are just renames.